charts/neon-proxy/values.yaml

```diff
@@ -26,8 +26,8 @@ settings:
   authEndpoint: ""
   # settings.uri - psql session endpoint, e.g. "http://console.neon/psql_session/"
   uri: ""
-  # settings.authBackend -- auth method used (legacy|console|link|postgres)
-  authBackend: "legacy"
+  # settings.authBackend -- auth method used (console|link|postgres)
+  authBackend: "link"
```

Comment:
Not directly related to this PR, but:

Can't we have this defined in just one repo? proxy-scram explicitly says that it needs `authBackend: "console"`, so why do we have to provide this value here? I find it very weird that we have to open coordinated PRs to at least two repos for such a simple thing as changing authBackend of a proxy instance.

Reply:
That's the first time I discovered how this helm hierarchy works for proxy. I think after explicitly specifying `authBackend` everywhere like this https://github.com/neondatabase/neon/pull/2452, we can remove default here